addon/services/tour.js

```diff
@@ -448,18 +448,20 @@ export default Service.extend(Evented, {
 
       let $window = $(window);
 
-      // Allow scrollbar scrolling so scrollTo works.
-      currentStep.options.scrollToHandler = (elem) => {
-        $window.disablescroll({
-          handleScrollbar: false
-        });
-
-        if (typeof elem !== 'undefined') {
-          elem.scrollIntoView();
-        }
+      if (currentStep.options.scrollToHandler === 'undefined') {
```

Comment:
Wouldn't this need to be typeof? Couldn't you just say `if (!currentStep.options.scrollToHandler)`